lua/nvim-treesitter/textobjects/select.lua

```diff
@@ -11,13 +11,86 @@ local M = {}
 function M.select_textobject(query_string, keymap_mode)
   local lookahead = configs.get_module("textobjects.select").lookahead
   local lookbehind = configs.get_module("textobjects.select").lookbehind
+  local include_surrounding_whitespace = configs.get_module("textobjects.select").include_surrounding_whitespace
   local bufnr, textobject =
     shared.textobject_at_point(query_string, nil, nil, { lookahead = lookahead, lookbehind = lookbehind })
   if textobject then
+    if include_surrounding_whitespace then
+      textobject = M.include_surrounding_whitespace(bufnr, textobject)
+    end
     ts_utils.update_selection(bufnr, textobject, M.detect_selection_mode(query_string, keymap_mode))
   end
 end
 
+function M.include_surrounding_whitespace(bufnr, textobject)
```

Comment:
all those added helper functions don't need to be public

Reply:
done